ui/app/utils/default_jobs/actions.js

```diff
@@ -0,0 +1,131 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+export default `job "redis-actions" {
+  datacenters = ["*"]
+
+  group "cache" {
+    count = 1
+    network {
+      port "db" {}
+    }
+
+    task "redis" {
+      driver = "docker"
+
+      config {
+        image = "redis:3.2"
+        ports = ["db"]
+        command = "/bin/sh"
+        args = ["-c", "redis-server --port \${NOMAD_PORT_db} & /local/db_log.sh"]
```

Comment:
Like above, we can probably use port mapping to keep this a bit cleaner and avoid the extra args, but if you want to use dynamic ports something like this would be a bit easier to understand?

```hcl
job "example" {
  group "cache" {
    network {
      port "db" {}
    }

    task "redis" {
      driver = "docker"

      config {
        image   = "redis:7"
        ports   = ["db"]
        args    = ["/etc/redis/redis.conf"]
        volumes = ["local/conf:/etc/redis"]
      }

      template {
        data        = <<EOF
port {{env "NOMAD_PORT_db"}}
EOF
        destination = "local/conf/redis.conf"
      }
    }
  }
}
```

Then the `redis-cli -p \${NOMAD_PORT_db} DBSIZE` command in `db_log.sh` could be another action?

Reply:
I realize it's not practical or desired for a real-world redis server on Nomad, but it seemed cool to illustrate db size output live when you make edits via action.

In the (pending) [actions tutorial](https://tutorials-git-1814-hashicorp.vercel.app/nomad/tutorials/job-specifications/job-spec-actions) ([PR](https://github.com/hashicorp/tutorials/pull/1814)), I hoped this would be something easily demonstrable to a user.